packages/client/hooks/useSelectTopTemplate.ts

```diff
@@ -1,10 +1,11 @@
 import {useEffect, useRef} from 'react'
+import {MeetingTypeEnum} from '../types/graphql'
 import {commitLocalUpdate} from 'react-relay'
-import SelectTemplate from '../mutations/SelectTemplateMutation'
 import isTempId from '../utils/relay/isTempId'
+import setTemplateId from '../utils/relay/setTemplateId'
 import useAtmosphere from './useAtmosphere'
 
-const useSelectTopTemplate = (edges: readonly {node: {id: string}}[], selectedTemplateId: string, teamId: string, isActive: boolean) => {
+const useSelectTopTemplate = (edges: readonly {node: {id: string}}[], selectedTemplateId: string, teamId: string, isActive: boolean, meetingType: MeetingTypeEnum) => {
```

Comment:
lets rename this since we're changing the active template & not the selected one.
maybe `useActivateTopTemplate`?